dotcom-rendering/src/client/decidePublicPath.ts

```diff
@@ -5,17 +5,9 @@
  */
 export const decidePublicPath = (): string => {
 	const isDev = process.env.NODE_ENV === 'development';
-	if (isDev && window.location.hostname === 'localhost') {
-		// Use the absolute path when in development mode and running locally
-		// Ensures asset paths are correct relative to Frontend
-		return 'http://localhost:3030/assets/';
-	} else if (
-		window.location.hostname === (process.env.HOSTNAME ?? 'localhost')
-	) {
-		// Use a relative path when hostname is set or using localhost in non-dev environments
-		// e.g. when using a reverse proxy, or running in CI
-		return '/assets/';
-	} else {
-		return `${window.guardian.config.frontendAssetsFullURL}assets/`;
-	}
+	const isLocalHost = window.location.hostname === 'localhost';
```

Comment:
Nice one!

Note to self: consider if we could always return `/assets/` and let the fact that the JS files are loaded from a different origin automatically handle their loading.